tests/probe/t15-specimen.sh

```
@@ -0,0 +1,66 @@
+# two well-separated regions so CodeRabbit opens a thread on each. Region A is
+# rewritten in a later commit to outdate its thread; region B is left untouched
+# and stays live. Delete with the branch.
+
+# ---------------------------------------------------------------- region A ---
+
+sum_sizes() {
+  DIR=$1
+  total=0
+  cd $DIR
+  for f in `ls $DIR`; do
+    sz=`cat $DIR/$f | wc -c`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)